dev_guide/templates.adoc

```diff
@@ -98,10 +98,9 @@ $ `osc process -f _<filename.json>_ | osc create -f -`
 ****
 
 You can override any link:../dev_guide/templates.html#parameters[parameters]
-defined in the JSON file by adding the `-v` option and any desired parameters.
-Note that the usage of the parameters is not restricted to only environment
-variables. You can use the parameter reference in any text field inside the
-template items.
+defined in the JSON file by adding the `-v` option followed by a
+comma-separated list of the form _<name>_=_<value>_.
+A parameter reference may appear in any text field inside the template items.
```

Comment:
If env-var only was the situation w/ OpenShift v2, then i think it would be good to add back the emphasis that such a restriction is no longer the case w/ v3.  If env-var only was instead just a temporary implementation detail or design cul-de-sac, then i don't think it warrants mentioning.
@mfojtik @adellape 
So, which was it?